src/core/abstract_image_file.h

```diff
@@ -16,7 +16,7 @@ class AbstractImageFile {
 
 	virtual bool IsOpen() = 0;
 
-	virtual bool OpenFile(std::string filename, bool overwrite = false, bool wait_for_file_to_exist = false, bool check_only_the_first_image = false) = 0; // Return true if everything about the file looks OK
+	virtual bool OpenFile(std::string filename, bool overwrite = false, bool wait_for_file_to_exist = false, bool check_only_the_first_image = false, int eer_super_res_factor = 1, int eer_frames_per_image = 0) = 0; // Return true if everything about the file looks OK
```

Comment:
Should eer_frames_per_image not be defaulted to 1?

Reply:
I left a comment about this in eer_file.h: 
`we default eer_frames_per_image to zero so that it'll be obvious if the user forgot to set it`

I'm basically worried that if the programmer forgets to set this value, then the file reader will go ahead and create 1 32-bit float frame per EER frame which would lead to enormous memory consumption, and likely a crash or big slow down.